apis/r/R/utils-tiledb.R

```diff
@@ -22,9 +22,10 @@ map_query_layout <- function(layout) {
     )
 }
 
-#' Display Package Versions
+#' Display package versions
```

Comment:
We may have used Title Style on purpose.  But we are possibly not quite consistent across so ...